frontend/src/components/TagList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+
+const TagList = props => {
+
+    const all_tags = props.tags.map(({ id, name }) => {
```

Comment:
```suggestion
    const AllTags = props.tags.map(({ id, name }) => {
```

this is component, it should be named with uppercase